theHarvester/discovery/dnssearch.py

```diff
@@ -1,8 +1,27 @@
+# -*- coding: utf-8 -*-
+
+"""
+============
+DNS Browsing
+============
+
+Explore the space around known hosts & ips for extra catches.
+"""
+
+import dns
+import re
 import sys
-import dns.resolver
+
+from aiodns import DNSResolver
+from ipaddress import IPv4Network
```

Comment:
Maybe not now but perhaps down the road there are instances where the tool grabs IPv6 addresses will there be any support for them?

Reply:
well technically it should be no problem, but I start from the ips returned by the search engines, still ipv4 so I didn't feel the need

Reply:
Suppose we grab "2001:db8:85a3:8d3:1319:8a2e:370:7348", what range would you reverse?

Reply:
/52 maybe?